checkstyle/import-control.xml

```diff
@@ -50,6 +50,9 @@
     <allow pkg="org.apache.kafka.common.internals" exact-match="true" />
     <allow pkg="org.apache.kafka.test" />
 
+    <allow pkg="org.openjdk.jmh.annotations" />
+    <allow pkg="org.openjdk.jmh.infra" />
```

Comment:
the benchmark is in the `org.apache.kafka.common.record` package. Now that we don't rely on package protected classes anymore, we can move it to `org.apache.kafka.jmh` and remove this.